### PR TITLE
fix(moveToLobby): need handle main locus info when move to lobby from breakout

### PR DESCRIPTION
**Pull request by mickelr**
# COMPLETES #< INSERT LINK TO ISSUE >

## This pull request addresses

When host move a web client to lobby in a breakout, the web client UI state should be right.

## by making the following changes

Need handle main session locus info in the case.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meetings/index.ts [hunk 1]
```diff
@@ -340,6 +342,9 @@ export default class Meetings extends WebexPlugin {
     if (newLocus) {
       const isNewLocusAsBreakout = MeetingsUtil.isBreakoutLocusDTO(newLocus);
       const isSelfMoved = newLocus?.self?.state === _LEFT_ && newLocus?.self?.reason === _MOVED_;
+      const isSelfMovedToLobby =
+        newLocus?.self?.devices[0]?.intent?.reason === _ON_HOLD_LOBBY_ &&
```
Comment (Coread) [inline packages/@webex/plugin-meetings/src/meetings/index.ts:346]: suggestion: you could use lodash's get here to provide a safe way of getting the reason without all those `?`
i.e.
```js
import {get} from 'lodash'

const isSelfMovedToLobby = get(newLocus, 'self.devices[0].intent.reason') === _ON_HOLD_LOBBY && 
get(newLocus, 'self.devices[0].intent.type') === _WAIT_; 

```

**Review by Coread — APPROVED** [review]
Please see suggestion regarding getting from object with all the `?`

**Review by mkesavan13 — APPROVED** [review]
Looks good to me. However, it'd be nice to attach the following in the PR description for future tracking:
- Issue Tracking Number
- Tests that were performed after the change
- Ticking the type of change in the description

File: packages/@webex/plugin-meetings/src/meetings/index.ts (hunk 1 above)
Comment (mickelr) [reply packages/@webex/plugin-meetings/src/meetings/index.ts:346]: Thanks for your suggestion

**Pull request merged**